contentScript.js

```diff
@@ -35,5 +46,8 @@ const interval = setInterval(() => {
     console.log("Hiding self...");
     clearInterval(interval);
     hideSelfButton().click();
+    setInterval(() => {
+      closeSendingVideoButton().click();
+    }, 2000);
```

Comment:
🔧 I feel 2 seconds might be a tad short - maybe 4 or 5 seconds would be better? That way it's less likely people will miss the message but not so long that it would get annoying, hopefully 😄 

```suggestion
    }, 4000);
```

Reply:
Sounds good, let's go with 4.